UPGRADE-1.8.md

```diff
@@ -19,22 +19,24 @@
 
     ```diff
         parameters:
-    +       sylius.security.new_api_admin_route: "/new-api"
-    +       sylius.security.new_api_admin_regex: "^%sylius.security.new_api_admin_route%"
+    +       sylius.security.new_api_route: "/new-api"
+    +       sylius.security.new_api_admin_route: "/new-api/admin"
+    +       sylius.security.new_api_admin_regex: "^/new-api/admin"
+    +       sylius.security.new_api_shop_regex: "^/new-api/(?!admin/.*)[^/]++"
```

Comment:
```suggestion
    +       sylius.security.new_api_shop_regex: "^/new-api/shop"
```
To be more explicit? We don't need to support different locales in path out of the box, I suppose